bitnami/mongodb/values-production.yaml

```diff
@@ -555,15 +652,23 @@ arbiter:
   ## extraFlags:
   ##  - "--wiredTigerCacheSizeGB=2"
   ##
-  extraFlags: []
+  ## extraFlags: []
+  extraFlags:
+  - "--tlsMode=requireTLS"
+  - "--tlsCertificateKeyFile=/certs/mongodb.pem"
+  - "--tlsCAFile=/certs/mongodb-ca-cert"
```

Comment:
I would do the same as proposed for the init container with these parameters.

Reply:
done